src/content/concepts/index.md

```diff
@@ -72,14 +72,15 @@ T> The `output` property has [many more configurable features](/configuration/ou
 
 *Loaders* enable webpack to process more than just JavaScript files (webpack itself only understands JavaScript). They give you the ability to leverage webpack's bundling capabilities for all kinds of files by converting them to valid [modules](/concepts/modules) that webpack can process.
 
-Essentially, webpack loaders transform all types of files into modules that can be included in your application's dependency graph.
+Essentially, webpack loaders transform all types of files into modules that can be included in your application's dependency graph (and eventually your bundle).
 
 W> Note that the ability to `import` any type of module, e.g. `.css` files, is a feature specific to webpack and may not be supported by other bundlers or task runners. We feel this extension of the language is warranted as it allows developers to build a more accurate dependency graph.
 
-At a high level, __loaders__ have two purposes in your webpack config. They work to:
+At a high level, __loaders__ have two purposes in your webpack configuration:
+
+1. The `test` property identifies which file or files should be transformed.
+2. The `use` property says which loader should be used to do the transforming.
 
```

Comment:
Love this change but two minor nitpicks:

- Can you change `says` in `2.` to `indicates`?
- Can you drop the additional blank line (`83`)? Typically we only include two line breaks before `h2` headers.